### PR TITLE
Implement ready system between rounds

### LINKED ISSUE #97: Ready system between rounds
**Description**
Create some kind of ready system between rounds.

**Subtasks**
- [x] All players have to ready up before next round.
- [x] UI Display

### PR DESCRIPTION
**Pull request by smeas**
Closes #97 

**List of changes**  
- Moved 'Start' input action to the 'UI' action map
- Implemented the ready system

**Additional context**  
image

### CODE REVIEW

File: Assets/00_Content/Scripts/Rounds/ScoreCard.cs [hunk 1]
```diff
@@ -18,17 +25,53 @@ public class ScoreCard : MonoBehaviour {
...
 			roundNumberText.text = RoundNumber.ToString();
 		}
 
-		public void GoToNextRound() {
+		public void Show() {
+			playerCount = PlayerManager.Instance.Players.Count;
+			for (int i = 0; i < readyCards.Length; i++) {
+				readyCards[i].gameObject.SetActive(i < playerCount);
+				readyCards[i].Ready = false;
+			}
```
Comment (MrJackey) [inline Assets/00_Content/Scripts/Rounds/ScoreCard.cs:53]: Just curious, couldn't this be combined with the loop below?

**Review by MrJackey — APPROVED** [review]
Nice, is this something we would want in the lobby aswell? :shipit:

File: Assets/00_Content/Scripts/Rounds/ScoreCard.cs (hunk 1 above)
Comment (smeas) [reply Assets/00_Content/Scripts/Rounds/ScoreCard.cs:53]: This loop is 0..MaxPlayers (see readyCards declaration), the other is 0..PlayerCount.

Comment (MrJackey) [reply Assets/00_Content/Scripts/Rounds/ScoreCard.cs:53]: Yes but is 0..Maxplayers needed? Is it for deactivating cards if a player suddenly disconnects or another reason?

Comment (smeas) [reply Assets/00_Content/Scripts/Rounds/ScoreCard.cs:53]: Hmm, yes. I suppose that was what I thought. But I guess that will probably never happen.

**Pull request merged**